package.json

```diff
@@ -15,7 +15,9 @@
         "typescript": "5.5.4"
     },
     "overrides": {
-        "@mantine/core@8.0.0>@mantine/hooks": "$@mantine-8/hooks"
+        "@mantine/core@8.0.0>@mantine/hooks": "$@mantine-8/hooks",
+        "axios": "^1.12.0",
+        "trino-client>axios": "^1.12.0"
```

Comment:
Required to fix axios dependency inside trino, until this is merged https://github.com/trinodb/trino-js-client/issues/893